beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -225,13 +328,69 @@ def normalize(tensor):
 plt.imshow(fbank.t().numpy(), cmap='gray')
 
 
+######################################################################
+# You can create mel frequency cepstral coefficients from a raw audio signal
+# This matches the input/output of Kaldi’s compute-mfcc-feats.
+# 
+
+mfcc = torchaudio.compliance.kaldi.mfcc(waveform, **params)
+
+print("Shape of mfcc: {}".format(mfcc.size()))
+
+plt.figure()
+plt.imshow(mfcc.t().numpy(), cmap='gray')
+
+######################################################################
+# Finally, you can resample your waveform at a new frequency, matching Kaldi’s
+# OfflineFeatureTpl ResampleWaveform, which uses a LinearResample.
+# 
+
+NEW_FREQ = sample_rate / 2
+resample_wf = torchaudio.compliance.kaldi.resample_waveform(waveform, sample_rate, NEW_FREQ)
+
+print("Shape of resample_wf: {}".format(resample_wf.size()))
+
+
```

Comment:
Thanks for exploring these, but we don't really want to advertise kaldi's compliance interface more than needed, so let's not include this.